### DISCUSSION TITLE
Launching an application before logging in G814JV 2023

### DISCUSSION THREAD

**Discussion by DripSweat** [seerge/g-helper#3307] [Q&A] [answered]
Good day to all.
"If this topic has already been discussed - please let me know (I didn't find a solution here)"
i use 0.191 version

The gist of the question:
By default, the application (g-helper) can be enabled to load at startup. However, this setting, according to the task scheduler, will launch the application only when the user has logged in. Has logged in correctly.

In view of the previously described question and problems, I tried to launch the application before logging in - I set the settings in the task scheduler. However, after setting the launch parameters to allow the application to launch before logging in, I got the following result:
1. The application launches correctly before logging in
2. When the user has logged in correctly - the application is not in the tray, but it exists and works in the processes.

Please help - how can I make the application launch before the user logs in and be displayed in the tray? (I tried many task scheduler settings - none helped solve the problem/question)

Thanks in advance!

**Comment by seerge** [comment, marked as answer]
@DripSweat latest releases (including your own) crate additional startup task that runs with windows (i.e. before user login). If it didn't happen to you automatically, you need to launch app at least once as admin so it can create this task.

**Reply by DripSweat** [reply]
Wow, thanks. I'll try it.

**Reply by DripSweat** [reply]
It works! It works! - Hooray!
